README.md

```diff
@@ -59,54 +69,114 @@ Some components or modules in forte may require some [extra requirements](https:
+            # second step: split stripped sentences into words and write
+            # words into the data pack
+            for w in words:
+                # we pass the input pack and word strings
+                # into `Word` class, and the `Word.value` store
+                # `input_pack` will internally add `Word` entries into `input_pack`'s internal storage
+                # while `Word` initializes itself.
+                Word(pack=input_pack, value=w)
+    # Configuration is a dictionary format
+    # it can be defined to hold user-input configurations
+    @classmethod
+    def default_configs(cls) -> Dict[str, Any]:
+        return {}
+```
+Finally, we set up pipeline and add all pipeline components into it, and process the input read from the terminal.
+```python
+pipeline: Pipeline = Pipeline[DataPack]()
+# add a reader that reads input by prompting user in the terminal
+pipeline.set_reader(TerminalReader())
+# NOTE: we can add multiple processors into the pipeline
+#       and pipeline processes the input sequentially without a conflict.
+
+# add the first processor: SpacyProcessor from third party library that extract entity mentions
+pipeline.add(SpacyProcessor(), {"processors": ["sentence", "ner"]})
+# add the second processor: the customized processor that split sentences into words
+pipeline.add(WordSplitPackProcessor())
+for pack in pipeline.initialize().process_dataset():
     for sentence in pack.get("ft.onto.base_ontology.Sentence"):
         print("The sentence is: ", sentence.text)
         print("The entities are: ")
         for ent in pack.get("ft.onto.base_ontology.EntityMention", sentence):
             print(ent.text, ent.ner_type)
-
+    print("Customized WordSplitPackProcessor results: ")
+    for token in pack.get(Word):
+        print(token, end = " ")
+    print()
 ```
 
 
-## Core Design Principles
+## Learn More
+With Forte, it is extremely simple to build an integrated system that can search documents, analyze, extract information and generate language all in one place.
+This allows developers to fully utilize the strength of individual modules, combine the results from each step, and enables the system to make fully informed decision at the end of the pipeline.
+
+Forte not only makes it easy to integrate with arbitrary 3rd party tools (Check out these [examples](https://github.com/asyml/forte/tree/master/examples)!), but also brings technology to you by offering a miscellaneous collection of deep learning modules via Texar, and a convenient model-data interface for casting tasks to models.
+
+
+
+* [Examples](https://github.com/asyml/forte/tree/master/examples)
+* [Documentation](https://asyml-forte.readthedocs.io/)
+* Currently we are working on some interesting tutorials such as [MT pipeline](https://github.com/asyml/forte/wiki) and [Text classification pipeline](https://asyml-forte.readthedocs.io/en/latest/notebook_tutorial/text_classification_pipeline.html)
+
+## Project Info
```

Comment:
I don't find this `Project Info` header useful

Reply:
It's because the headers under this would be too redundant on the left bar if we keep the original header levels. https://asyml-forte--730.org.readthedocs.build/en/730/